index.html

```diff
@@ -18,10 +19,46 @@
                 </ul>
                 <i class="fa-solid fa-bars fa-xl" style="color: #0f0f10;"></i>
             </div>
-
-            
         </nav>
     </div>
+
+    <!--section-->
+
+    <div class="hero-section">
+        <img src="./image/head shot.jpg" alt="">
+
+    </div>
+        
+    
+    <h2>Hi I am Soko,<br>      
+    creative <br> Technologist</h2>
+
+    <div class="text">
+        <p>Stay away from those people who try to disparge your ambitions.Small minds will always do that,but great minds will give you a feeling that you can become great too.-Mark Twain
+        </p>
+        <button
+         class="download"download>
+         <a href="download">Download Resume</a>
+         </button>
+         
+        
+    </div>
```

Comment:
Weldone, you are progressing. Please continue working on the project and remember that you re to follow the mobile-first approach.

I have nothing else to say to you other than you should merge your project to the main branch.

<img src="https://media1.giphy.com/media/XD4BoRtenzE1eTIHzZ/giphy.gif"/>